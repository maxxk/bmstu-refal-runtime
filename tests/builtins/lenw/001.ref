$ENTRY Go
{
	= <Prout <Lenw 'asd' ASd sd 213>>
}
