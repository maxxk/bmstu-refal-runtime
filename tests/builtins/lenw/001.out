6 asdASd sd 213 
